main/templates/allauth/account/login.html

```diff
@@ -16,20 +16,39 @@
         <div class="card-action teal lighten-2 white-text">
           <h3>Login</h3>
         </div>
-
+        {% if form.errors %}
```

Comment:
Line 19 to 32 display the errors to the users...like wrong username/password and so .